src/features/session_trace/aggregate/index.js

```diff
@@ -443,7 +443,11 @@ export class Aggregate extends AggregateBase {
       this.storeResources(window.performance.getEntriesByType('resource'))
     }
 
+    let earliestTimeStamp = Infinity
     const stns = Object.entries(this.trace).flatMap(([name, listOfSTNodes]) => { // basically take the "this.trace" map-obj and concat all the list-type values
+      const oldestNodeTS = listOfSTNodes.reduce((acc, next) => (!acc || next.s < acc) ? next.s : acc, undefined)
```

Comment:
btw you can also start with `0` and not have to check acc, I think?

Reply:
then nothing would ever be less than 0

Reply:
oops I meant `Infinity` heh